src/settings/JobProfiles/JobProfilesForm.js

```diff
@@ -40,6 +40,8 @@ const formName = 'jobProfilesForm';
 const dataTypes = DATA_TYPES.map(dataType => ({
   value: dataType,
   label: dataType,
+  // TODO: Disabling options should be removed after implentation is done
+  disabled: dataType === 'EDIFACT',
```

Comment:
Consider using a constant here

```suggestion
  disabled: dataType === DATA_TYPES.EDIFACT,
```

Reply:
IT won't work here. 
![image](https://user-images.githubusercontent.com/40805351/95732864-0000ba80-0c8a-11eb-92bc-b8e0a41a7a32.png)